js/src/forum/components/Category.tsx

```diff
@@ -141,10 +156,16 @@ export default class Category extends Component<Attrs> {
   alignStartItems() {
     const items = new ItemList();
 
-    const tag = this.tag;
-    const children = this.isChild ? [] : sortTags(tag.children() || []);
-
-    items.add('icon', <span className="TagCategory-icon">{this.iconItems().toArray()}</span>, 100);
+    const children = this.isChild ? [] : sortTags(this.tag.children() || []);
+    let style: Record<string, unknown> = {};
+    if (this.isChild) {
```

Comment:
This "style building" code seems to be repeated; could you extract it into a helper function?